template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hue/HueConfigProvider.java

```diff
@@ -138,7 +138,6 @@ private String getCdhVersionString(TemplatePreparationObject source) {
 
     private boolean externalFQDNShouldConfigured(GatewayView gateway, GeneralClusterConfigs generalClusterConfigs) {
         return gateway != null
-                && generalClusterConfigs.getKnoxUserFacingCertConfigured()
```

Comment:
Why was this removed?

Reply:
If it is not removed, and there is a Let's Encrypt outage, Hue knox proxy hosts property is not set. The common workaround for Let's Encrypt outages is to let the cluster be created and users are asked to click the renew certificate button. Unfortunately `generalClusterConfigs.getKnoxUserFacingCertConfigured()` is not set to true for those clusters. This means Hue will become unusable via load-balancer.